### PR TITLE
Add links, FAQ questions

### PR DESCRIPTION
**Pull request by jhepkema**
Because it's an important part of the design of NEO.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,7 +34,8 @@
 ## Blockchain Bootstrap Files
 
 - [MainNet (drive.google.com)](https://drive.google.com/open?id=0B902ABEfKU8YTUg4ckFXTldRczA) - Up to around block 1350000.
-- [MainNet (mega.nz)](https://mega.nz/#!JHQFgbYa!TVygP7EIZonl42i_Tpx4CV_26b1CEgPoOU04dq10h8E) - Up to around block 1427475.
+- [MainNet (mega.nz)|chain.acc.zip](https://mega.nz/#!JHQFgbYa!TVygP7EIZonl42i_Tpx4CV_26b1CEgPoOU04dq10h8E) - Up to around block 1427475.
```
Comment (revett) [inline README.md]: Use following format:

```
[MainNet (mega.nz - chain.acc.zip)](https://mega.nz/#!JHQFgbYa!TVygP7EIZonl42i_Tpx4CV_26b1CEgPoOU04dq10h8E) - Up to around block 1427475.
```

(same for link below as well)

File: resources/dapps.md [hunk 2]
```diff
@@ -84,22 +84,39 @@ When listing a dApp, stick to the following guidelines:
...
+  - **Team**: See the team at [adex.network](http://adex.network/).
+  - **Status**: In Development
+  - **Links**: [Github Page](https://github.com/AdExBlockchain), [Medium](https://medium.com/the-adex-blog), [Facebook](https://www.facebook.com/AdEx-by-Stremio-756186427900381/), [Telegram](https://t.me/AdExNetwork), [Twitter](https://twitter.com/AdEx_Network), [Reddit](https://www.reddit.com/r/AdEx/).
+
+- **Project Name**: Red Pulse Token
+  - **Short Description**: Red Pulse Tokens (RPX) are NEO tokens issued by Red Pulse, an event-driven research firm covering market events impacting Chinese companies, sectors and the overall economy.
+  - **Team**: See the team at [coin.red-pulse.com](https://coin.red-pulse.com).
+  - **Status**: ICO planned for October 8 2017 at:
```
Comment (revett) [inline resources/dapps.md]: Can this be updated?

File: resources/faq.md [hunk 3]
```diff
@@ -1,5 +1,12 @@
 # FAQ
 
+---
```
Comment (revett) [inline resources/faq.md]: No need for this `---`.

**Comment by jhepkema** [comment]
Thanks for the feedback @revett, applied it.

**Comment by jhepkema** [comment]
Also, I'm busy with [this](https://github.com/neo-project/docs/pull/194) (translating everything from docs.neo.org to Dutch); do you know anyone to mention in there so it can get reviewed by someone who speaks Dutch?

**Comment by jhepkema** [comment]
I also thought of adding the Aphelion ICO but currently I cannot find enough data on them.

**Comment by revett** [comment]
@jhepkema When the Dutch translation is complete, I'll pass it over to CoZ developers.

**Comment by jhepkema** [comment]
Thanks! I will tag you in the PR once it's done :)  
Thanks for the review!

**Pull request merged**